TypeScript.YAML-tmLanguage

```diff
@@ -26,11 +26,18 @@ repository:
       '1': { name: variable.ts }
     end: (?=$|[;,])
     patterns:
+    - include: '#ternary-operator'
     - include: '#type-annotation'
     - include: '#string'
     - include: '#comment'
     - include: '#expression'
 
+  ternary-operator:
```

Comment:
('?' and ':') together are known as the ternary operator, not sure if we should call it expression or operator. Also note that we end at ':', which can be followed by an expression (since expressions can exist anywhere.)